apps/web/src/routes/AuthLayoutRouter/components/ContextMenu/ContextMenu.jsx

```diff
@@ -415,7 +634,12 @@ function ListItemRenderer ({ item, rootPath, groupSlug, canDnd, isOverlay = fals
       <DropZone hide={hideDropZone || invalidChild || !canDnd} droppableParams={{ id: `${item.id}`, data: { widget: item } }}>
         &nbsp;
       </DropZone>
-      <li ref={setItemDraggableNodeRef} style={itemStyle} className='flex justify items-center content-center animate-slide-up invisible'>
+      <li
+        ref={setItemDraggableNodeRef}
+        style={itemStyle}
+        className='flex justify items-center content-center animate-slide-up invisible'
+        data-view-id={item.id}
```

Comment:
seems a bit weird to be using data-view-id on two different elements, one div and one li, what is it needed on both for?